subgraph.yaml

```diff
@@ -51,6 +51,52 @@ dataSources:
         - event: URI(string,indexed uint256)
           handler: handleURI
       file: ./src/mapping.ts
+  - kind: ethereum/contract
+    name: SuperRare
+    network: mainnet
+    source:
+      address: "0x41a322b28d0ff354040e2cbc676f0320d8c8850d"
+      abi: SuperRare
+      startBlock: 5364090
+    mapping:
+      kind: ethereum/events
+      apiVersion: 0.0.4
+      language: wasm/assemblyscript
+      entities:
+        - WhitelistCreator
+        - Bid
+        - AcceptBid
+        - CancelBid
+        - Sold
+        - SalePriceSet
+        - OwnershipTransferred
+        - Transfer
+        - Approval
+      abis:
+        - name: SuperRare
+          file: ./abis/SuperRare.json
+        - name: ERC721
+          file: ./abis/ERC721.json
+      eventHandlers:
+        - event: WhitelistCreator(indexed address)
+          handler: handleWhitelistCreator
+        - event: Bid(indexed address,indexed uint256,indexed uint256)
```

Comment:
As I see, those events are not used for now. Maybe it will be better to add them here later if we gonna use them, so the code would be cleaner now

Reply:
The graph framework creates the eventHandlers by default. I like it when I can see all events without looking in the solidity contract. That's why I let it in, but I can remove it.

Reply:
Got it, let's leave it for now.
But in general, I recommend to use a proper tool for a proper job: there should be some explorers that will show you all the events of the contract.